torch/testing/_internal/common_methods_invocations.py

```diff
@@ -1002,6 +1030,19 @@ def sample_inputs_pinverse(op_info, device, dtype, requires_grad=False):
                                 dtypes=[torch.bfloat16])),
                    promotes_integers_to_float=True,
                    handles_complex_extremals=False),
+    OpInfo('linalg.slogdet',
```

Comment:
How time consuming are the grad and gradgrad checks for slogdet on CPU and CUDA?

Reply:
Before this commit b3f8932c27:
```
39.03s call     test/test_ops.py::TestGradientsCUDA::test_fn_gradgrad_linalg_slogdet_cuda_complex128
14.70s call     test/test_ops.py::TestGradientsCPU::test_fn_gradgrad_linalg_slogdet_cpu_complex128
10.48s call     test/test_ops.py::TestGradientsCUDA::test_fn_gradgrad_linalg_slogdet_cuda_float64
4.61s call     test/test_ops.py::TestGradientsCUDA::test_fn_grad_linalg_slogdet_cuda_complex128
3.81s call     test/test_ops.py::TestGradientsCPU::test_fn_gradgrad_linalg_slogdet_cpu_float64
2.33s call     test/test_ops.py::TestOpInfoCUDA::test_supported_dtypes_linalg_slogdet_cuda_complex128
2.12s call     test/test_ops.py::TestGradientsCUDA::test_fn_grad_linalg_slogdet_cuda_float64
1.72s call     test/test_ops.py::TestGradientsCPU::test_fn_grad_linalg_slogdet_cpu_complex128
0.75s call     test/test_ops.py::TestGradientsCPU::test_fn_grad_linalg_slogdet_cpu_float64
```
and after:
```
4.84s call     test/test_ops.py::TestGradientsCUDA::test_fn_gradgrad_linalg_slogdet_cuda_complex128
1.91s call     test/test_ops.py::TestOpInfoCUDA::test_supported_dtypes_linalg_slogdet_cuda_complex128
1.88s call     test/test_ops.py::TestGradientsCPU::test_fn_gradgrad_linalg_slogdet_cpu_complex128
1.54s call     test/test_ops.py::TestGradientsCUDA::test_fn_grad_linalg_slogdet_cuda_complex128
1.33s call     test/test_ops.py::TestGradientsCUDA::test_fn_gradgrad_linalg_slogdet_cuda_float64
0.51s call     test/test_ops.py::TestGradientsCPU::test_fn_gradgrad_linalg_slogdet_cpu_float64
0.39s call     test/test_ops.py::TestGradientsCUDA::test_fn_grad_linalg_slogdet_cuda_float64
0.22s call     test/test_ops.py::TestGradientsCPU::test_fn_grad_linalg_slogdet_cpu_complex128
```